packages/backend-for-frontend/src/model/errors.ts

```diff
@@ -426,3 +428,13 @@ export function delegationNotFound(delegationId: string): ApiError<ErrorCodes> {
     title: "Delegation not found",
   });
 }
+
+export function delegatedEserviceNotExportable(
+  delegatorId: string
+): ApiError<ErrorCodes> {
+  return new ApiError({
+    detail: `Impossibile to export Eservice with a valid delegation for producer ${delegatorId}`,
+    code: "delegationWithEserviceIdFound",
+    title: "EService delegated",
```

Comment:
```suggestion
    code: "delegatedEserviceNotExportable",
    title: "Delegated Eservice is not exportable",
```
The code and title with this change should be more understandable